src/index.html

```diff
@@ -8,9 +8,49 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      class="form"
+      data-qa="big"
+    >
+
+      <label class="form-field form-field-big">
```

Comment:
there should be no blank lines between parent and child tags
```suggestion
    >
      <label class="form-field form-field-big">
```